package-lock.json

```diff
@@ -1,19 +1,18876 @@
 {
     "name": "kusto",
     "version": "0.0.12",
-    "lockfileVersion": 1,
+    "lockfileVersion": 2,
```

Comment:
I noticed you weren't pushing changes to the lock file so I put it in a separate commit that I can revert